dev/pages/TextField.vue

```diff
@@ -39,13 +36,10 @@ const moneyMask = { numeral: true, numeralPositiveOnly: true, numeralIntegerScal
       </w-textfield>
     </token>
 
-    <!-- TODO tabbing through the input and suffix is off for now. We do not have support for adding multiple slots as of now. So both
-    suffix and prefix are treated as one slot in this example, making button(suffix) render before input in DOM and "destroy" the tabbing order.
-    So the support for multiple slots need to be added here -->
     <token :state="inputModel">
-      <w-textfield #prefix #suffix v-model="inputModel" label="I have a prefix" inputmode="numeric">
-        <w-affix prefix label="+47" />
-        <w-affix suffix clear />
+      <w-textfield v-model="inputModel" label="I have a prefix and a suffix" inputmode="numeric" input-wrapper-class="[&]:pl-4!">
```

Comment:
What does `[&]` stand for here?

Reply:
Well, that's... just leftover crap. 🙈 😅  Thanks! Good catch! 🙏  🗑️